.gitlab-ci.example.yml

```diff
@@ -16,15 +16,17 @@
 # https://docs.gitlab.com/ee/ci/yaml/
 
 .dependabot:
-  image: dependabot/dependabot-core
+build-image:
+  stage: build
```

Comment:
Does this automatically get built and run before the `.dependabot` step?

Reply:
Yes, it does.

Reply:
Tested successfully on a GitLab instance.

Reply:
Hm, but line 18. now should be deleted yet.

Reply:
Thanks for confirming, fixed the stray line 👍